packages/docs/docs/guide/markdown.md

```diff
@@ -121,6 +121,9 @@ Rendering of TOC can be configured using the [`markdown.toc`](../config/README.m
 
 ## Custom Containers <Badge text="default theme"/>
 
+Containers can be defined by their types, titles, and contents. The default titles would be applied while customized titles were missing or empty.
+
+### Apply Default Titles
```

Comment:
Since this is the default behavior, we can restructure this to just be an examples section of the custom containers and how they work.

```suggestion
**Examples**
```